activities/Gwen.py

```diff
@@ -0,0 +1,160 @@
+#!/usr/bin/env python3
+
+import os
+
+import grass.script as gs
+
+import numpy as np
+
```

Comment:
Found this in tangible_utils and you use it later.

```suggestion
def get_environment(**kwargs):
    """!Returns environment for running modules.
    All modules for which region is important should
    pass this environment into run_command or similar.

    @param tmp_regions a list of temporary regions
    @param kwargs arguments for g.region

    @return environment as a dictionary
    """
    env = os.environ.copy()
    env["GRASS_OVERWRITE"] = "1"
    env["GRASS_VERBOSE"] = "0"
    env["GRASS_MESSAGE_FORMAT"] = "standard"
    region3d = False
    if "raster_3d" in kwargs:
        region3d = True
    env["GRASS_REGION"] = gs.region_env(region3d=region3d, **kwargs)
    return env

```